app/services/image_blob_service.rb

```diff
@@ -3,10 +3,11 @@
 require "mini_magick"
 
 class ImageBlobService < ApplicationService
-  def initialize(revision, user, temp_image)
+  def initialize(temp_image:, revision: nil, user: nil, hardcoded_filename: nil)
+    @temp_image = temp_image
     @revision = revision
     @user = user
-    @temp_image = temp_image
+    @hardcoded_filename = hardcoded_filename
```

Comment:
If I understand right we're using this input as a way to skip the unique file service which I think we'll actually need. So I think we can consider removing this.